### ISSUE TITLE
Workaround for "Error: Error opening /dev/sda: No medium found"

### ISSUE THREAD

**Issue by philon123** [WoeUSB/WoeUSB#113]
Trying to use this tool while a USB is mounted will fail (there are many bug reports / feature requests open on that already). So it's necessary to unmount the USB before using the tool. 

However, if I use Nautilus on Ubuntu 22.10 to unmount the USB, then woeusb usage fails:

```
WoeUSB v5.2.4
==============================
Info: Mounting source filesystem...
Info: Wiping all existing partition table and filesystem signatures in /dev/sda...
Info: Ensure that /dev/sda is really wiped...
Info: Creating new partition table on /dev/sda...
Error: Error opening /dev/sda: No medium found
ERROR: The command "parted --script "${target_device}" mklabel "${parted_partiton_table_argument}"" failed with exit status "1", program is prematurely aborted
Info: Unmounting and removing "/tmp/woeusb-source-20221030-115717-Sunday.ZeFfP4"...
Info: You may now safely detach the target device
```

After some googling, it seems there is a difference between unmounting via Nautilus vs via command line. So this ticket is to document that you need to unmount the USB via command line, not Nautilus:
`sudo umount /media/philon/bla` (using your user and mounted volume name of course) 

I hope this shows up in someones search results and saves their time!

**Comment by corndogit** [comment]
Yes, this solved the issue for me also - I was trying to unmount the drive through Nemo on Linux Mint 21.1 and getting the same error

**Comment by sanchit-lamba** [comment]
thanks for the post it worked as intended

**Comment by calebmeyer** [comment]
To add to this the full list of steps that worked for me:
- unplug the now "half unmounted" drive
- plug it back in
- use your file manager or another means to get the /media name (for me it was `/run/media/caleb/USB\ Disk`) on Fedora workstation
- `sudo umount /run/media/caleb/USB\ Disk` -- note: this is `umount`, not `unmount`
- `sudo ./woeusb-5.2.4.bash --device ./Win10_22H2_English_x64v1.iso /dev/sda`

**Comment by Vel1khan** [comment]
Thank you, this worked
